docs/reference/asset-reference.md

```diff
@@ -124,27 +117,24 @@ The `download_stac_asset()` function allows you to download STAC assets from sto
 ```python
 download_stac_asset(
     stac_asset,
-    unpacking
+    output_directory,
 )
 ```
-<h5> Arguments </h5>
 
-| Name               | Type                     | Description                |
-| ------------------ | ------------------------ | -------------------------- |
-| `stac_asset`       | `pystac.Asset`           | The STAC asset name.       |
-| `output_directory` | `Union[str, Path, None]` | The file output directory. |
+The returned format is `Path`.
 
-<h5> Returns </h5>
+<h5> Arguments </h5>
 
-| Type   | Description                               |
-| ------ | ----------------------------------------- |
-| `Path` | The path where the file were uploaded to. |
+| Argument           | Description                                                                                           |
+| ------------------ | ----------------------------------------------------------------------------------------------------- |
+| `stac_asset`       | **pystac.Asset / required**<br/>The STAC asset name.                                                  |
+| `output_directory` | **Union[str, Path, None]**<br/>The file output directory. The default value is the current directory. |
```

Comment:
`The default value is the current directory.`
This is partially correct. We create some additional sub directories in the current directory:
`output_directory = (
                Path.cwd()
                / f"project_{self.auth.project_id}/asset_{self.asset_id}/{stac_asset.title}"
            )`

Reply:
But you're creating new folders in all cases, regardless of whether an output directory is provided or not, correct?

Reply:
yes

Reply:
I'll add this to the description:
"A new directory for the file will be created."